src/org/labkey/test/tests/SampleTypeLinkedStudyExportTest.java

```diff
@@ -0,0 +1,148 @@
+
+@Category({Daily.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 2)
+public class SampleTypeLinkedStudyExportTest extends BaseWebDriverTest
+{
+    private final static String SAMPLE_TYPE_PROJECT = "Sample Type Project";
+    private final static String SAMPLE_TYPE = "Samples";
+    private final static String LINKED_STUDY = "Target Study Project";
+    private final static String IMPORT_PROJECT = "Import Test Project";
+    private static TestDataGenerator SAMPLE_GENERATOR;
+
+    protected DateTimeFormatter _dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    protected String now = LocalDateTime.now().format(_dateTimeFormatter);
+
+    @BeforeClass
+    public static void setupProject() throws IOException, CommandException
+    {
+        SampleTypeLinkedStudyExportTest init = (SampleTypeLinkedStudyExportTest) getCurrentTest();
+        init.doSetup();
+    }
+
+    @Override
+    protected @Nullable String getProjectName()
+    {
+        return SAMPLE_TYPE_PROJECT;
+    }
+
+    private void doSetup() throws IOException, CommandException
+    {
+        _containerHelper.createProject(getProjectName(), null);
+
+        _containerHelper.createProject(LINKED_STUDY, "Study");
+        _studyHelper.startCreateStudy()
+                .setTimepointType(StudyHelper.TimepointType.VISIT)
+                .createStudy();
+        String containerId = ((APIContainerHelper) _containerHelper).getContainerId(LINKED_STUDY);
+
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        log("Creating sample types");
+        SAMPLE_GENERATOR = new SampleTypeDefinition(SAMPLE_TYPE)
+                .setAutoLinkDataToStudy(containerId)
+                .setFields(List.of(
+                        new FieldDefinition("VisitID", FieldDefinition.ColumnType.VisitId),
+                        new FieldDefinition("Date", FieldDefinition.ColumnType.VisitDate),
+                        new FieldDefinition("ParticipantID", FieldDefinition.ColumnType.Subject)))
+                .create(createDefaultConnection(), SAMPLE_TYPE_PROJECT);
+
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        new PortalHelper(getDriver()).addBodyWebPart("Sample Types");
+
+        _containerHelper.createProject(IMPORT_PROJECT, "Study");
+    }
+
+    /*
+        Test coverage for : Issue 45238: NPE when importing sample type with auto-link study via pipeline job
+     */
+    @Test
+    public void testImportSampleTypeFolder()
+    {
+        goToProjectHome(SAMPLE_TYPE_PROJECT);
+        clickAndWait(Locator.linkWithText(SAMPLE_TYPE));
+        DataRegionTable samplesTable = DataRegionTable.DataRegion(getDriver()).withName("Material").waitFor();
+        samplesTable.clickInsertNewRow();
+        setFormElement(Locator.name("quf_Name"), "Blood");
+        setFormElement(Locator.name("quf_VisitID"), "1");
+        setFormElement(Locator.name("quf_Date"), now);
+        setFormElement(Locator.name("quf_ParticipantID"), "P1");
+        clickButton("Submit");
```

Comment:
FYI, `clickInsertNewRow` returns a page object you could use.
```suggestion
        samplesTable.clickInsertNewRow().update(Map.of(
            "Name", "Blood"
            "VisitID", "1"
            "Date", now
            "ParticipantID", "P1"));
```